pkg/cmd/build/view.go

```diff
@@ -57,8 +56,14 @@ func NewCmdBuildView(f *factory.Factory) *cobra.Command {
 				return fmt.Errorf("could not resolve a build")
 			}
 
+			if web {
+				buildUrl := fmt.Sprintf("https://buildkite.com/%s/%s/builds/%s", bld.Organization, bld.Pipeline, args[0])
```

Comment:
args won't necessarily have a value here. I think this should be `bld.BuildNumber`?